atlasdb-config/src/main/java/com/palantir/atlasdb/config/TimeLockRequestBatcherProviders.java

```diff
@@ -28,4 +29,6 @@ public interface TimeLockRequestBatcherProviders {
     TimeLockRequestBatcherProvider<MultiClientTransactionStarter> startTransactions();
 
     TimeLockRequestBatcherProvider<MultiClientCommitTimestampGetter> commitTimestamps();
+
+    TimeLockRequestBatcherProvider<MultiClientTimeLockUnlocker> unlock();
```

Comment:
nit: this is a Java API break as at least one consumer will need to update to provide an instance here

```
[ImmutablesBuilderMissingInitialization] Some builder fields have not been initialized: unlock
ERROR: [ImmutablesBuilderMissingInitialization] Some builder fields have not been initialized: unlock
                .build();
                      ^
    (see https://github.com/palantir/gradle-baseline#baseline-error-prone-checks)
```